assets/pages/support-landing/supportLanding.jsx

```diff
@@ -44,17 +49,19 @@ const content = (
         headings={['Help us deliver', 'the independent', 'journalism the', 'world needs']}
         highlights={['Support', 'The Guardian']}
       />
-      <Contribute
-        copy="Your contribution funds and supports The Guardian's journalism."
-      >
-        <ContributionSelectionContainer />
-        <ContributionPaymentCtasContainer
-          PayPalButton={PayPalContributionButtonContainer}
-        />
-      </Contribute>
-      <ThreeSubscriptions />
+      <section id={supporterSectionId}>
+        <Contribute
+          copy="Your contribution funds and supports The Guardian's journalism."
+        >
+          <ContributionSelectionContainer />
+          <ContributionPaymentCtasContainer
+            PayPalButton={PayPalContributionButtonContainer}
+          />
+        </Contribute>
+        <ThreeSubscriptions />
+      </section>
       <WhySupport />
-      <ReadyToSupport ctaUrl="#" />
+      <ReadyToSupport ctaUrl={`#${supporterSectionId}`} />
```

Comment:
The argument is called ctaUrl but an id is being passed. Do we want to pass an URL or a css selector?

Reply:
That's a [URL fragment](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-href), linking to further up the page.